backend/upload/gretel4/cgn.json

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
Do these json files belong in the repo? Where is this information stored for the other corpora in gretel?